src/test/skript/tests/syntaxes/effects/EffToggleCanPickUpItems.sk

```diff
@@ -0,0 +1,7 @@
+test "entity can pick up items":
+	spawn a zombie at spawn of "world":
+		allow event-entity to pick up items
+		assert event-entity can pick up items with "failed to allow zombie to pick up items"
+		forbid event-entity from picking items up
+		assert event-entity can't pick up items with "failed to disallow zombie to pick up items"
+		delete event-entity
```

Comment:
newline at end of file. This is old programming standard to let the programming language know the coding has ended, most modern langauges can tell when it's the end of the file. GitHub complains about no new line at the end of the file. So it's better to adapt to this standard.
```suggestion
		delete event-entity

```